src/resources/2017/Spacegate.ts

```diff
@@ -0,0 +1,155 @@
+import { cliExecute, Item, visitUrl } from "kolmafia";
+import { get } from "../../property";
+import { $item, $items } from "../../template-string";
+
+export function have(): boolean {
+  return get("spacegateAlways") || get("_spacegateToday");
+}
+
+export function checkStatus(): void {
```

Comment:
`updateStatus` might be a more informative name for something with return type `void`